### DISCUSSION TITLE
Transparency

### DISCUSSION THREAD

**Discussion by rubiin** [Jas-SinghFSU/HyprPanel#339] [Q&A]
How to have similar transparency

![image](https://github.com/user-attachments/assets/d9715eb8-ea38-423e-9665-5a493738ed79)

![image](https://github.com/user-attachments/assets/bae35138-8e90-4efb-8067-6287bd281fcd)

**Comment by Jas-SinghFSU** [comment]
You can adjust your bar and menu opacity in:
1. Settings > Theming > General Settings > Menu Opacity
2. Settings > Theming > Bar > Various Opacity Options

**Reply by rubiin** [reply]
I could not quite achieve the blurry effect as shown .

**Reply by Jas-SinghFSU** [reply]
That you would have to do via hyprland settings.

**Reply by rubiin** [reply]
let me try

**Reply by sayidabyan** [reply]
menu opacity would change all elements like text's opacity too tho right? I think normally people only wants to change the background (maybe the button as well) opacity instead of everything.

**Comment by NinthJake** [comment]
From another commenter I found that you can enable the blurred look for the bar itself by adding this layerrule to your hyprland config `layerrule = blur, bar-.*` but I don't know how to enable the same effect for the dashboard or other menus. 
Also by lowering transparency on the dashboard in the hyprpanel settings it makes everything transparent, including the buttons, profile picture and the text. It would be great to be able to modify the transparency of only the menu background.
